src/styles/components/_about-us-introduction-section.scss

```
@@ -0,0 +1,197 @@
+.about-us-introduction-section {
```

`section` is a `section` while `introduction` is a block. Don't combine them when you can make them into a separate component.
```suggestion
.about-us-introduction {
  display: flex;
  flex-direction: column;
  justify-content: space-between;
  height: rem(250px);

  @include respond-above('sm') {
    height: rem(400px);
  }

  @include respond-above('md') {
    height: rem(500px);
  }
  
  @include respond-above('lg') {
    height: rem(600px);
  }

  @include respond-above('xl') {
    height: rem(700px);
  }
  
  &_title {// props}
  
  &__description {// props}
  
  &--background {
    background: {
      image: url(../../../public/images/hand-shake.jpg);
      repeat: no-repeat;
      position: left bottom 35%;
      size: 50% auto;
    }
  }
}

.about-us-section {
  // some shareable props with other sections
}
```

---

You can also do this to align the description at the bottom right.
```
&__description {
    align-self: flex-end;

    box-sizing: border-box;
    width: 55%;
    color: $light;
    background-color: black;
    padding: 4% 7% 4% 2.5%;
}
```